### ISSUE TITLE
TypeError: memoizeOne is not a function

### ISSUE THREAD

**Issue by TheEhsanSarshar** [microlinkhq/metascraper#432]
I am using metascraper in Expo ( a favor of React native ).
after running and building the project on the web. this issue arises.

```
TypeError: memoizeOne is not a function
node_modules/@metascraper/helpers/index.js:280
```

**Comment by Kikobeats** [comment]
Can you do a `console.log(require('memoize-one'))` to see what's is returning?

**Comment by TheEhsanSarshar** [comment]
after changing ```const memoizeOne = require('memoize-one')``` to ```const memoizeOne = require('memoize-one').default``` issue fixed

**Comment by Kikobeats** [comment]
Thanks, that was my suspicion.

Released `metascraper@5.22.9` if you install it should be work fine now 🙂

**Comment by TheEhsanSarshar** [comment]
@Kikobeats thank you.
I was going to make a PR and then I found that you just released a new version.
